src/components/Books.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Books = () => {
+  <div>
+    <h3>Title</h3>
+    <p>Author</p>
+  </div>;
+};
+
```

Comment:
This is the follow-up for the above comment I left on the `BookList` component.
1. The reason the books are not showing up on the screen is because they never make up to the `Books` component. Since you are passing the information to the `Books` component in the `BookList` component, `Books` component expects you to pass in the `props` argument which will contain all the information passed to the `Books` component. From there you will have to de-structure the info you want.
2. This component is returning nothing. That is the main reason nothing is showing on the screen. You should return the div explicitly as ever react component should return something.